src/header/Header.tsx

```diff
@@ -1,83 +1,76 @@
-// @ts-check
+/* eslint-disable react/require-default-props */
 import React from 'react';
-import PropTypes from 'prop-types';
 import { getConfig } from '@edx/frontend-platform';
 import { useIntl } from '@edx/frontend-platform/i18n';
 import { StudioHeader } from '@edx/frontend-component-header';
 import { useToggle } from '@openedx/paragon';
 
-import SearchModal from '../search-modal/SearchModal';
+import { SearchModal } from '../search-modal';
 import { getContentMenuItems, getSettingMenuItems, getToolsMenuItems } from './utils';
 import messages from './messages';
 
+interface HeaderProps {
+  contentId?: string,
```

Comment:
```suggestion
  contextId?: string,
```
The generic term for "course or library" is "learning context" so for consistency this parameter should be `contextId` not `contentId`

Reply:
Done: c92b6b6c88950f7b110c7c2e7849bdec28b9621f